[2025-02-18T11:05:13.042+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T11:05:13.070+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_python_operator_v7.get_name scheduled__2025-01-22T00:00:00+00:00 [queued]>
[2025-02-18T11:05:13.087+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_python_operator_v7.get_name scheduled__2025-01-22T00:00:00+00:00 [queued]>
[2025-02-18T11:05:13.087+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-02-18T11:05:13.115+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_name> on 2025-01-22 00:00:00+00:00
[2025-02-18T11:05:13.126+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_python_operator_v7', 'get_name', 'scheduled__2025-01-22T00:00:00+00:00', '--job-id', '456', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmprxqnz88m']
[2025-02-18T11:05:13.133+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=217) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T11:05:13.134+0000] {standard_task_runner.py:72} INFO - Started process 224 to run task
[2025-02-18T11:05:13.134+0000] {standard_task_runner.py:105} INFO - Job 456: Subtask get_name
[2025-02-18T11:05:13.223+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_python_operator_v7.get_name scheduled__2025-01-22T00:00:00+00:00 [running]> on host 063e621c401a
[2025-02-18T11:05:13.423+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sandeep' AIRFLOW_CTX_DAG_ID='dag_python_operator_v7' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-01-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-22T00:00:00+00:00'
[2025-02-18T11:05:13.424+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-18T11:05:13.522+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T11:05:13.549+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-18T11:05:13.550+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_python_operator_v7, task_id=get_name, run_id=scheduled__2025-01-22T00:00:00+00:00, execution_date=20250122T000000, start_date=20250218T110513, end_date=20250218T110513
[2025-02-18T11:05:13.698+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T11:05:13.743+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-18T11:05:13.748+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
